docs/cog_guides/admin.rst

```diff
@@ -0,0 +1,435 @@
+
+^^^^^^^
+addrole
+^^^^^^^
+
+.. note:: |admin-lock| This is also usable by the members with the ``Manage
+    roles`` permission.
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]addrole <rolename> [user]
+
+**Description**
+
+Adds a role to a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to add to the member. |role-input-quotes|
+
+* ``[user]``: The member you want to add the role to. Defaults to the
+  command author. |member-input|
+
+.. _admin-command-removerole:
+
+^^^^^^^^^^
+removerole
+^^^^^^^^^^
+
+.. note:: |admin-lock| This is also usable by the members with the
+    ``Manage roles`` permission.
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]removerole <rolename> [user]
+
+**Description**
+
+Removes a role from a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to remove. |role-input-quotes|
+
+* ``[user]``: The member to remove the role from. |member-input| Defaults
+    to the command author.
+
+.. _admin-command-editrole:
+
+^^^^^^^^
+editrole
+^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole
+
+**Description**
+
+Edits the settings of a role.
+
+.. _admin-command-editrole-name:
+
+"""""""""""""
+editrole name
+"""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole name <role> <name>
+
+**Description**
+
+Edits the name of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input-quotes|
+
+* ``<name>``: The new role name.
+
+.. _admin-command-editrole-color:
+
+""""""""""""""
+editrole color
+""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole color <role> <color>
+
+**Description**
+
+Edits the color of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input-quotes|
+
+* ``<color>``: The new color to assign. |color-input|
+
+**Examples**
+
+* ``[p]editrole color "My role" #ff0000``
+
+* ``[p]editrole color "My role" dark_blue``
+
+.. _admin-command-announce:
+
+^^^^^^^^
+announce
+^^^^^^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announce <message>
+
+**Description**
+
+Announce your message to all of the servers the bot is in.
+
+The bot will announce the message in the guild's announcements channel.
+If this channel is not set, the message won't be announced.
+
+**Arguments**
+
+* ``<message>``: The message to send.
+
+.. _admin-command-announce-cancel:
+
+"""""""""""""""
+announce cancel
+"""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announce cancel
+
+**Description**
+
+Cancels an active announcement.
+
+.. _admin-command-announceset:
+
+^^^^^^^^^^^
+announceset
+^^^^^^^^^^^
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announceset
+
+**Description**
+
+Change how announcements are recieved in this guild.
```

Comment:
```suggestion
Change how announcements are received in this guild.
```
*I before E except after C*

Reply:
*or when sounds like A as in neighbor or weigh*

*or whenever English feels like it because English is w**ei**rd*